### DISCUSSION TITLE
Gif library?

### DISCUSSION THREAD

**Discussion by Sirablopp** [mastodon/mastodon#20136] [General]
I recently noticed a lack of an easy way to share gifs via an embedded gif library. (Should there be one and I missed it, please tell me where to find it.) Uploading gifs as a file works, but I am afraid my hard disk is not big enough for storing every gif there is on the internet. ;) But a discussion platform of any kind without gifs just feels like a nice tool for further communication is missing. So.. can we please have some?

**Comment by bosh-ahmed** [comment]
I 2nd this


Alot of the ‘regular’ people dipping their toes in Mastodon struggle with gifs, specially as the bird app has searchable gifs built in..

**Comment by Sirablopp** [comment]
From what I read people manually uploading the gifs they are used to also requires a lot of server space. Wouldn't it be better if people had an option to post a link to the gif in question instead of uploading it? Or would a gif be rehosted on a fediverse server regardless?
